extensions/sql-migration/src/wizard/migrationModePage.ts

```diff
@@ -25,115 +30,99 @@ export class MigrationModePage extends MigrationWizardPage {
 
 		const pageDescription = {
 			title: '',
-			component: view.modelBuilder.text().withProps({
-				value: constants.DATABASE_BACKUP_MIGRATION_MODE_DESCRIPTION,
-				CSSStyles: {
-					...styles.BODY_CSS,
-					'margin': '0'
-				}
-			}).component()
+			component: view.modelBuilder.text()
+				.withProps({
+					value: constants.DATABASE_BACKUP_MIGRATION_MODE_DESCRIPTION,
+					CSSStyles: { ...styles.BODY_CSS, 'margin': '0' }
+				}).component()
 		};
 
 		const form = view.modelBuilder.formContainer()
-			.withFormItems(
-				[
-					pageDescription,
-					this.migrationModeContainer(),
-				]
-			).withProps({
-				CSSStyles: {
-					'padding-top': '0'
-				}
-			}).component();
+			.withFormItems([
+				pageDescription,
+				this.migrationModeContainer()])
+			.withProps({ CSSStyles: { 'padding-top': '0' } })
+			.component();
+
+		this._disposables.push(
+			this._view.onClosed(
+				e => this._disposables.forEach(
+					d => { try { d.dispose(); } catch { } })));
 
-		this._disposables.push(this._view.onClosed(e => {
-			this._disposables.forEach(
-				d => { try { d.dispose(); } catch { } });
-		}));
 		await view.initializeModel(form);
 	}
 
 	public async onPageEnter(pageChangeInfo: azdata.window.WizardPageChangeInfo): Promise<void> {
-		this.originalMigrationMode = this.migrationStateModel._databaseBackup.migrationMode;
-		this.wizard.registerNavigationValidator((e) => {
-			return true;
-		});
+		const isSqlDbTarget = this.migrationStateModel._targetType === MigrationTargetType.SQLDB;
+		this._onlineButton.enabled = !isSqlDbTarget;
```

Comment:
Can we add some text/info box/tooltip that explains why online migration is not available for SQL DB?